src/envoy/http/mixer/filter.cc

```diff
@@ -259,8 +229,10 @@ void Filter::log(const HeaderMap* request_headers,
     CheckData check_data(*request_headers, nullptr);
     handler_->ExtractRequestAttributes(&check_data);
   }
-  ReportData report_data(response_headers, request_info, sent_bytes_,
-                         received_bytes_);
+  response_total_size_ =
```

Comment:
we don't need response_total_size as member variable. 
please add comment
// response trailer header is not counted to response total size.

Reply:
Done. Thanks.